unit-tests/live/d400/test-hdr.py

```diff
@@ -0,0 +1,575 @@
+        depth_sensor.set_option(rs.option.sequence_size, 2)
+        test.check(depth_sensor.get_option(rs.option.sequence_size) == 2)
+
+        depth_sensor.set_option(rs.option.sequence_id, 1)
+        test.check(depth_sensor.get_option(rs.option.sequence_id) == 1)
+        depth_sensor.set_option(rs.option.exposure, 120)
+        test.check(depth_sensor.get_option(rs.option.exposure) == 120)
+        depth_sensor.set_option(rs.option.gain, 90)
+        test.check(depth_sensor.get_option(rs.option.gain) == 90)
+
+        depth_sensor.set_option(rs.option.sequence_id, 2)
+        test.check(depth_sensor.get_option(rs.option.sequence_id) == 2)
+        depth_sensor.set_option(rs.option.exposure, 1200)
+        test.check(depth_sensor.get_option(rs.option.exposure) == 1200)
+        depth_sensor.set_option(rs.option.gain, 20)
+        test.check(depth_sensor.get_option(rs.option.gain) == 20)
+
+        depth_sensor.set_option(rs.option.hdr_enabled, 1)
+        test.check(depth_sensor.get_option(rs.option.hdr_enabled) == 1)
+
+        depth_sensor.set_option(rs.option.hdr_enabled, 0)
+        test.check(depth_sensor.get_option(rs.option.hdr_enabled) == 0)
+
+        device.hardware_reset()
+        time.sleep(5)
+
+
+# HDR STREAMING TESTS
+with test.closure("HDR Streaming - default config"):
+    device = test.find_first_device_or_exit()
+    depth_sensor = device.first_depth_sensor()
+
+    if depth_sensor and depth_sensor.supports(rs.option.hdr_enabled):
+        exposure_range = depth_sensor.get_option_range(rs.option.exposure)
+        gain_range = depth_sensor.get_option_range(rs.option.gain)
+
+        depth_sensor.set_option(rs.option.hdr_enabled, 1)
+        test.check(depth_sensor.get_option(rs.option.hdr_enabled) == 1)
+
+        cfg = rs.config()
+        cfg.enable_stream(rs.stream.depth)
+        cfg.enable_stream(rs.stream.infrared, 1)
+        pipe = rs.pipeline()
+        pipe.start(cfg)
+
+        for iteration in range(1, 100):
+            data = pipe.wait_for_frames()
+            out_depth_frame = data.get_depth_frame()
+            if iteration < 3:
+                continue
+
+            if out_depth_frame.supports_frame_metadata(rs.frame_metadata_value.sequence_id):
+                frame_exposure = out_depth_frame.get_frame_metadata(rs.frame_metadata_value.actual_exposure)
+                frame_gain = out_depth_frame.get_frame_metadata(rs.frame_metadata_value.gain_level)
+                seq_id = out_depth_frame.get_frame_metadata(rs.frame_metadata_value.sequence_id)
+
+                if seq_id == 0:
+                    test.check(frame_exposure == exposure_range.default - 1000)  # w/a
+                    test.check(frame_gain == gain_range.default)
+                else:
+                    test.check(frame_exposure == exposure_range.min)
+                    test.check(frame_gain == gain_range.min)
+        pipe.stop()
+        depth_sensor.set_option(rs.option.hdr_enabled, 0)  # disable hdr before next tests
+
+
```

Comment:
No reset between these tests - is it by purpose? If so, please explain, otherwise, please add.

Reply:
Yes it's on purpose, by [your comment](https://github.com/IntelRealSense/librealsense/pull/12670/files/d895e06b95b8fee0809b3bc2bb185b1443637c14#r1494162998) I figured it's not needed here?